features/contain.yml

```diff
@@ -0,0 +1,11 @@
+name: contain
+description: "The `contain` CSS property sets limits to the scope of styles, layout, and paint rendering for speed and efficiency. The `none` keyword value disables containment. The `strict` and `content` keyword values are equivalent to `contain: size layout style paint` and `contain: layout style paint` respectively."
+spec: https://drafts.csswg.org/css-contain-2/#contain-property
+group: containment
+caniuse: css-containment
+compat_features:
+  # Already included in `contain: size`
+  #- css.properties.contain
```

Comment:
It's fine to move keys around as needed, and I think it makes sense to move this here.